proposals/NNNN-PerformInteractionMultipick.md

```diff
@@ -0,0 +1,249 @@
+So, add a field `multiPickConfirmationButtonText`, if this has text in it then the `performInteraction` will be multipick. If the InteractionMode is VR_ONLY or the displayLayout is `TILES_WITH_SEARCH`, `LIST_WITH_SEARCH`, or `KEYBOARD_ONLY`, then the response will be INVALID_DATA. If the InteractionMode is BOTH, then the VR portion of the PerformInteraction will be skipped.
+
+
+Add `multiPickConfirmationButtonText` to the request in the `MOBILE_API`.
+Also, add a return parameter `choiceIDArray` to `PerformInteraction` response. This array allows duplicate choice IDs so that the HMI can send back the same choice ID multiple times to relay a quantity back to the app:
+```xml
+    <function name="PerformInteraction" functionID="PerformInteractionID" messagetype="request" since="1.0">
+      <description>Triggers an interaction (e.g. "Permit GPS?" - Yes, no, Always Allow).</description>
+      .
+      .
+      .
+      <param name="multiPickConfirmationButtonText" type="String" maxlength="500" mandatory="false" since="X.X">
+        <description>
+          If this parameter is set, then this is a Multipick performInteraction. This text will be in the button that a person uses to finish choosing items.
+        </description>
+      </param>
+    </function>
+	
+    <function name="PerformInteraction" messagetype="response">
+    .
+    .
+    .
+      <param name="choiceIDArray" type="Integer" minsize="1" maxsize="100" minvalue="0" maxvalue="2000000000" array="true" mandatory="false" since="X.X">
+       <description>
+          IDs of the choices that were selected in response to a multipick performInteraction. Allows duplicate choiceIDs so that the HMI can send back the same choice ID multiple times to relay a quantity back to the app.
+          Only is valid if general result is "success:true".
+       </description>
+      </param>	
+    </function>	
+```
+
+Then do the same for the `HMI_API`:
+```xml
+<interface name="UI" version="X.X.X" date="X-X-X">
+.
+.
+.
+  <function name="PerformInteraction" messagetype="request">
+  .
+  .
+  .
+    <param name="multiPickConfirmationButtonText" type="String" maxlength="500" mandatory="false">
+      <description>
+        If this parameter is set, then this is a Multipick performInteraction. This text will be the button that a person uses to finish choosing items.
+      </description>
+    </param>
+  </function>
+  
+  <function name="PerformInteraction" messagetype="response">
+  .
+  .
+  .
+    <param name="choiceIDArray" type="Integer" minsize="1" maxsize="100" minvalue="0" maxvalue="2000000000" array="true" mandatory="false">
+      <description>
+        IDs of the choices that were selected in response to a multipick performInteraction. Allows duplicate choiceIDs so that the HMI can send back the same choice ID multiple times to relay a quantity back to the app.

```

Comment:
```suggestion
        IDs of the choices that were selected in response to a multipick performInteraction. Allows duplicate choiceIDs so that the HMI can send back the same choiceID multiple times to relay a quantity back to the app.
```